docs/deployment/cumulus-distribution.md

```diff
@@ -20,15 +20,19 @@ To configure a deployment to use Cumulus Distribution:
  4. Rename `cumulus-template-deploy/cumulus-tf/cumulus_distribution.tf.example` to `cumulus-template-deploy/cumulus-tf/cumulus_distribution.tf`.
  5. Set the Cumulus Distribution variables in your `terraform.tfvars` (which is based on [terraform.tfvars.example](https://github.com/nasa/cumulus-template-deploy/blob/master/cumulus-tf/terraform.tfvars.example)). These include:
 
-- deploy_cumulus_distribution
-- cumulus_distribution_url
-- csdap_client_id
-- csdap_client_password
-- csdap_host_url
+- `deploy_cumulus_distribution`: Set to `true` if deploying the Cumulus Distribution API or `false` if deploying TEA.
+- `cumulus_distribution_url`: Used to override the base URL for your distribution API that will be inserted in CMR metadata. This can be used if you need to insert a port in your distribution URL for port forwarding or specify a Cloudfront URL for distribution. Not all users will need to set this and, if you do require it, it will need to be set _after_ an initial deployment. The process would be:
+  1. Deploy once without `cumulus_distribution_url` set
+  2. If necessary, redeploy with the correct value a `cumulus_distribution_url`. For port forwarding/tunnel based access, you might set `cumulus_distribution_url = "https://abc123.execute-api.us-east-1.amazonaws.com:7000/dev/". If you want to direct distribution to a Cloudfront URL that you registered for your API Gateway, you would set that for `cumulus_distribution_url`: `cumulus_distribution_url = https://abc123.cloudfront.net`.
+- `csdap_client_id`: The Client ID of your AWS account's Cognito setup, which is created and managed outside of Cumulus.
+- `csdap_client_password`: The client password for your AWS account's Cognito setup, which is created and managed outside of Cumulus.
+- `csdap_host_url`: The host URL of your AWS account's Cognito setup, which is created and managed outside of Cumulus.
```

Comment:
Although these values are managed outside of Cumulus, I strongly recommend adding some sort of note here indicating who should be contacted in order to obtain these values.

The "who" shouldn't be a specific person, but rather, which team, specifically the team who would know what a user is referring to when asking about how to obtain these values for a Cumulus deployment.

Otherwise, someone reading these documents is left wondering (blocked on) how to obtain these values and who to contact to obtain them.